src/applications/pensions/config/form.js

```diff
@@ -1544,7 +1548,11 @@ const formConfig = {
           path: 'additional-information/contact',
           uiSchema: {
             'ui:title': 'Contact information',
-            veteranAddress: address.uiSchema('Mailing address'),
+            veteranAddress: _.set(
+              'ui:validations[1]',
```

Comment:
How is this used?

Reply:
This is adding the central mail zip code validation to the list of validation functions, which already has one function for the general address validation.